p2p/issues/pubsub_349/main.go

```diff
@@ -0,0 +1,160 @@
+package main
```

Comment:
do we need this committed?

Reply:
yes, it is a proof of the issue and a boiler plate for future issues (not to rewrite most of the code over and over and over again)